docs/en/admins/06_LinuxInstall.md

```diff
@@ -69,6 +84,29 @@ Finally, symlink the public folder to the root of your web directory
 ln -s /usr/share/FreshRSS/p /var/www/html/
 ```
 
-### TODO: setup MySQL database for FreshRSS
+## Part 3: Creating a Database for FreshRSS
+
+Start a MySQL session. running this command will ask you for the MySQL password you set earler, and then put you into a prompt that should look like `MariaDB [(none)]>`
+```
+mysql -u root -p
+```
+
+From the MySQL prompt (`MariaDB [(none)]>`), run the following commands, substituting \<username>, \<password>, and \<database_name> for real values.
+```
+CREATE USER '<username>'@'localhost' IDENTIFIED BY '<password>';
+CREATE DATABASE `databaseName`;
+GRANT ALL privileges ON `databaseName`.* TO 'userName'@localhost;
+FLUSH PRIVILEGES;
+QUIT;
+```
+
+A brief explination of the previous command block:
+* You first create a database user for FreshRSS to use.
+* Then you create a database for FreshRSS to store data in.
+* You grant permissions for the user you created to read, write, and modify the database.
+* Flushing privileges reloads the permissions, making the previous command to take effect.
```

Comment:
typo

Reply:
I don't see the typo here :sweat_smile:

Reply:
*To* take effect is ungrammatical, at least in a subordinate clause like this.
```suggestion
* Flushing privileges reloads the permissions, which makes the previous command take effect.
```

Reply:
Ah, I knew I should have listened in class!